src/vs/workbench/contrib/interactiveEditor/browser/interactiveEditorActions.ts

```diff
@@ -547,3 +548,20 @@ export class ContractMessageAction extends AbstractInteractiveEditorAction {
 		ctrl.updateExpansionState(false);
 	}
 }
+
+export class AccessibilityHelpEditorAction extends EditorAction2 {
+	constructor() {
+		super({
+			id: 'interactiveSession.action.accessibilityHelpEditor',
+			title: localize('actions.interactiveSession.accessibiltyHelpEditor', "Interactive Session Editor Accessibility Help"),
```

Comment:
Maybe use `AbstractInteractiveEditorAction.category` and keep the title shorter and/or extend from `AbstractInteractiveEditorAction`